addons/quickmount/functions/fnc_getInNearest.sqf

```diff
@@ -0,0 +1,48 @@
+/*
+ * Author: Kingsley
+ * Mount the given player in the nearest eligible vehicle based on their distance and direction to the vehicle
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call acex_quickmount_fnc_getInNearest;
+ *
+ * Public: No
+ */
+
+#include "script_component.hpp"
+
+if (!GVAR(enabled)) exitWith {};
+if (isNull ACE_player || {vehicle ACE_player != ACE_player} || {!alive ACE_player}) exitWith {};
+
+private _start = AGLtoASL positionCameraToWorld [0,0,0];
+private _end = (_start vectorAdd (getCameraViewDirection ACE_player vectorMultiply GVAR(distance)));
+private _objects = lineIntersectsWith [_start, _end, ACE_player, objNull, true];
```

Comment:
why you dont use https://community.bistudio.com/wiki/lineIntersectsSurfaces? it would be faster and you could save the next line